### DISCUSSION TITLE
Badges and buttons (Marketing Campaigns / SendGrid)

### DISCUSSION THREAD

**Discussion by banand86** [twilio-labs/paste#1462] [General] [answered]
Hey team!
I need some help with badge UI in MC. I've see this "NEW" badge (view image) used in the product. However, it looks different from badges shown in the [SG styleguide](https://styleguide.sendgrid.com/misc.html).

- Which badge should I use?
- What are some guidelines around usage and placement (in UI) of the badges? Attached is an image that shows 3 options (in button, out of button, and overlapping button)
<img width="1166" alt="Screen Shot 2021-05-17 at 5 35 36 PM" src="https://user-images.githubusercontent.com/80920953/118569288-dc798300-b736-11eb-83bc-f4c10da64c1d.png">

Thanks for your help! :)

**Comment by SiTaggart** [comment, marked as answer]
Hi @banand86 

We can give you a little historical based information and general guidance. 

First general guidance regardless of whether it's Styleguide or Paste; Badges are usually best placed _after_ the content they are describing, so we suggest placing the badge after the text and inside the button.

For the historical information, typically something that is "new" in sendgrid is depicted with the green badge. You can see that as an example in the styleguide link you provided. Historically the purple is reserved for beta features, which I believe is also the case for Twilio products, so some good alignment there.

As for which do you use, Paste or Styleguide, that's up to you and the product team to decide right now, and also the context of the work being carried out.

**Reply by banand86** [reply]
Awesome! Thanks for the prompt response. This was helpful :)
